packages/qwik-city/middleware/cloudflare-pages/api.md

```diff
@@ -7,7 +7,7 @@
 import type { ServerRenderOptions } from '@builder.io/qwik-city/middleware/request-handler';
 
 // @public (undocumented)
-export function createQwikCity(opts: QwikCityCloudflarePagesOptions): (ctx: EventPluginContext) => Promise<Response>;
+export function createQwikCity(opts: QwikCityCloudflarePagesOptions): (eventPluginContext: EventPluginContext) => Promise<Response>;

```

Comment:
I've renamed this to `eventPluginContext` just to try not to use `ctx` since that could be confusing as is the same name as a user facing field we want to expose